testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeRedirect.java

```diff
@@ -877,6 +877,24 @@ public void testViewReferencingHiveAndDeltaTable(boolean legacyHiveViewTranslati
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testDeltaToHivePropertiesRedirect()
```

Comment:
This functionality has worked also before this PR.

We should be testing the other way around

`test_redirect_to_delta_properties`

Reply:
Modified both tests to verify redirection to the Delta table $properties table.